README.md

```diff
@@ -125,7 +125,7 @@ The `docs-content` class (the `.` means this is a class) is the main container o
 
 Every searchable `lvl` elements outside this main documentation container (for instance, in a sidebar) must be `global` selectors. They will be globally picked up and injected to every document built from your page.
 
-If your website is a VuePress application, you can check out the [config file](https://github.com/meilisearch/documentation/blob/master/.vuepress/docs-scraper/docs-scraper.config.json) we use in production.<br>
+If your website is a VuePress application, you can check out the [config file](https://github.com/meilisearch/documentation/blob/master/.vuepress/docs-scraper/docs-scraper.<path-to-your-config-file>) we use in production.<br>
```

Comment:
The link is broken. This shouldn't be updated :)

Reply:
Oh :sweat_smile: My bad!